ushahidi/platform-client
# Updating notification-message

This pull request makes the following changes:
-

Testing checklist:
- create a new targeted survey with many questions and many recipients and publish it
- [ ] After publishing, a notification should appear with the message "<the total number of messages > messages will be sent as part of this survey. You will receive an invoice for the total cost to these messages via email"
- create another targeted survey with one question and one recipient and publish it.
- [ ] - [ ] After publishing, a notification should appear with the message "1 message will be sent as part of this survey. You will receive an invoice for the total cost to that message via email"

Fixes ushahidi/platform# .

Ping @ushahidi/platform
